quodlibet/po/fr.po

```diff
@@ -2760,17 +2759,16 @@ msgid "The duplicate songs will be removed from the playlist '%s'."
 msgstr ""
 
 #: ../quodlibet/ext/playlist/shuffle.py:15
-#, fuzzy
 msgid "Shuffle Playlist"
-msgstr "Créer une liste de lecture"
+msgstr "Lecture aléatoire des titres"
```

Comment:
The Shuffle Playlist plugin actually shuffles tracks in the playlist, not just play in random order, so something more like "Mélanger la liste de lecture" ..